pages/nextstory.js

```diff
@@ -0,0 +1,13 @@
+import Card from '../components/Card';
+
+export default function nextStory() {
+  return (
+    <>
+      <Card>
+        ALICE was beginning to get very tired of sitting by her sister on the
+        bank, and of having nothing to do: once or twice she had peeped into the
+        book her sister was reading..
+      </Card>
```

Comment:
```suggestion
      <Card/>
```
If you import and call a component on your page, if should already display the content from the component (as you put in Card.js) so there is no need for another Text.

Reply:
done:)